eng/service-templates/ProductConstructionService/production.bicepparam

```diff
@@ -0,0 +1,57 @@
+using 'provision.bicep'
+
+param location = 'westus2'
+
+param containerRegistryName = 'productconstructionprod'
+
+param containerCpuCoreCount = '1.0'
+
+param containerMemory = '2Gi'
+
+param aspnetcoreEnvironment = 'Production'
+
+param applicationInsightsName = 'product-construction-service-ai-prod'
+
+param keyVaultName = 'ProductConstructionProd'
+
+param devKeyVaultName = 'ProductConstructionDev'
```

Comment:
Is this needed here?

Reply:
Fixed, it's not, we only create the Dev keyvault if the `environment` argument is `Staging`